packages/staking/package.json

```
@@ -33,18 +32,16 @@
   ],
   "scripts": {
     "build": "tsc && tsup",
-    "build:dev": "tsc & tsup",
     "cleanup": "yarn exec rm -rf dist node_modules && yarn exec rm -rf ../../.cache/eslintcache",
-    "dev": "ladle serve",
+    "dev": "start-storybook -p 6006",
     "eslint:check": "yarn run -T eslint --cache --cache-location ../../.cache/eslintcache --cache-strategy metadata --ignore-path '../../.eslintignore' '**/*.{cjs,mjs,js,ts,tsx}'",
     "eslint:fix": "yarn eslint:check --fix",
     "lint": "echo 'Staking package runs lint in its own CI'",
     "lint:all": "yarn eslint:check && yarn prettier:check",
     "lint:fix": "yarn eslint:fix && yarn prettier:fix",
     "prettier:check": "yarn run -T prettier --check '**/*.{js,ts,jsx,tsx,html,json,md}'",
     "prettier:fix": "yarn run -T prettier --write --loglevel warn '**/*.{js,ts,jsx,tsx,html,json,md}'",
-    "story:build": "ladle build",
-    "story:preview": "ladle preview",
+    "story:build": "build-storybook",
```

would i make sence to have same steps as in the browser-extension app?
```suggestion
        "storybook": "ENV=development start-storybook -p 6006",
    "prestorybook:build": "yarn build",
    "storybook:build": "ENV=development build-storybook --log-level error",
```

---

I've updated scripts now we have it `build` and `build:storybook`. I hope it makes more sense now. `pre*` step in this case would be a killer. I cannot imagine building all packages each time I run storybook. Everyone has to do it manually if needed 🙏🏻